extension/observer/endpoints_test.go

```diff
@@ -228,6 +232,54 @@ func TestEndpointEnv(t *testing.T) {
 				"labels": map[string]string{
 					"label_key": "label_val",
 				},
+				"host": "127.0.0.1",
+				"port": "1234",
```

Comment:
This is the test case that ensures the `Target` gets properly parsed as `host` and `port`.